### ISSUE TITLE
IDs formatted as floats

### ISSUE THREAD

**Issue by smythp** [JonathanReeve/corpus-db#24]
The IDs when getting subject metadata all have .0 at the end. Seems like they should be integers, not floats.

**Comment by JonathanReeve** [comment]
Yep indeed. I bet this would be an easy fix, and language-agnostic. It'd just need to be fixed in the DB and in the API documentation.

**Comment by smythp** [comment]
Yes, I could probably do this one.

**Comment by JonathanReeve** [comment]
Oh, and this would need to be fixed in the `data/dev.db`, too. If you do this, let me know what command/script you use, so I can replicate it on my local copy of the DB.

**Comment by smythp** [comment]
Yep no promises but if I have time I'll do it. It's a relatively simple fix but it's a big data set :)
